### ISSUE TITLE
Development: APIDocs warning

### ISSUE THREAD

**Issue by munsterlander** [opendatacam/opendatacam#476]
When you do a build, you get this warning:

```
{"message":"parser plugin 'param' not found in block: 1 in file: /home/mark/ODC/opendatacam/.build/apidoc/main.js","level":"warn"}
```

This is because the output folder is included in the input folder while generating the api docs during the build process as documented here:  https://github.com/apidoc/apidoc/issues/672

We need to update: `/scripts/npm/generateapidoc.sh` https://github.com/munsterlander/opendatacam/blob/development/scripts/npm/generateapidoc.sh line 7 `npx apidoc -i $REPO_ROOT -e node_modules/ -o .build/apidoc/`, but I am uncertain of the downstream impact if I just change the paths.  Can anyone provide clarification on what it should be so we do not get this warning or are we ok with it?

**Comment by vsaw** [comment]
Good Catch! Did you try what happens when we exclude .build from the APIdoc generation (like we do with node_modules)

**Comment by munsterlander** [comment]
I didn't yet because I wasn't certain what the dependencies were.  I can give it a go in a bit.

**Comment by munsterlander** [comment]
@vsaw So the code is in PR: https://github.com/opendatacam/opendatacam/pull/475 which excludes the directory and fixes the issue.  Additionally, with all the changes, now all the warnings are gone from the build process.  I added a comment at the bottom of that PR, but do you prefer one commit per PR or slam them all together like I did?  I wasn't really paying attention until the end there - apologies if that's now how you want it done.

**Comment by vsaw** [comment]
Is like to keep this open until it is fixed in Development. If you don't mind.

**Comment by munsterlander** [comment]
No worries at all.

**Comment by vsaw** [comment]
Closed via #493
